sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFnReflector.java

```diff
@@ -82,31 +117,32 @@
       .putAll(EXTRA_CONTEXTS)
```

Comment:
EXTRA_CONTEXTS = contexts available in startBundle and finishBundle (currently none)
EXTRA_PROCESS_CONTEXTS = contexts available in processElement

EXTRA_PROCESS_CONTEXTS should be a superset of EXTRA_CONTEXTS, so we add all of them.

No longer applicable -- take a look?